docs/user-guide/rasa-tutorial.rst

```diff
@@ -223,8 +223,23 @@ different model parts if something has changed in their data or configuration.
 
 The ``rasa train`` command will look for both NLU and Core data and will train a combined model.
 
+7. Test the model
+^^^^^^^^^^^^^^^^^
 
-7. Talk to Your Assistant
+After you train a model, you always want to check that your assistant still behaves as you expect.
+In Rasa, you use end-to-end tests defined in your ``tests/`` directory to run through
+test conversations that ensure both NLU and Core make correct predictions.
+
+.. runnable::
+
+   rasa test
+
+   echo "Finished running tests."
+
+See :ref:`evaluating-models` to find out how you can define more tests and learn about more ways you
```

Comment:
what do you mean by "how you can define more tests? I'd probably just say `to learn more about how to evaluate your model as you improve it`

Reply:
v good point, stole your copy